book/SUMMARY.md

```diff
@@ -31,6 +23,17 @@
 
 # Zapp Plugins
 
+## Zapp Plugin Abilities
```

Comment:
I would add abilities Under types , not above

Reply:
Done